src/NHibernate/Util/CollectionHelper.cs

```diff
@@ -214,6 +214,12 @@ public IEnumerator GetEnumerator()
 
 		public static readonly IEnumerable EmptyEnumerable = new EmptyEnumerableClass();
 		public static readonly IDictionary EmptyMap = new EmptyMapClass();
+
+		public static IDictionary<TKey, TValue> EmptyMapFor<TKey, TValue>()
```

Comment:
Rename to `EmptyMap` or `EmptyDictionary`

Reply:
@hazzik Done. I've also moved Instance to separate class - `EmptyDictionaryHolder<TKey, TValue>`. As `EmptyMapClass<TKey, TValue>` is public class and it's better to avoid unnecessary static initialization if it wasn't requested explicitly.